### DISCUSSION TITLE
Sujet / Thema 1 –– Einfacherer Zugang zu Geodaten / Accès plus facile aux géodonnées

### DISCUSSION THREAD

**Discussion by rastrau** [GeoUnconference/discussions#3] [Ideen / Idées]
Resultate der Diskussion an der GeoUnconference 1 / Résultats de la discussion lors de la GeoUnconference 1

![Access to geodata via a central platform](https://user-images.githubusercontent.com/4510815/124478349-6512b800-dda5-11eb-9da2-2df833d38a74.jpg)

![Access to geodata (2)](https://user-images.githubusercontent.com/4510815/124478354-66dc7b80-dda5-11eb-868a-4d6c75a2c695.jpg)

![Access to geodata](https://user-images.githubusercontent.com/4510815/124478383-6ba12f80-dda5-11eb-91a9-6ed0b272bb27.jpg)

**Comment by m-kuhn** [comment]
Interessante Diskussion.

Ich bin gerade nicht ganz sicher, ob das hier der richtige Ort ist ... Aber ich versuchs Mal.

Beim Herumstöbern auf geocat.ch und der Suche nach "gewaesserschutz" kommen verschiedene kantonale Resultate, gemischt mit schweizweiten. Diese sind teilweise auch einzeln mit "WMS" und "WFS" gekennzeichnet.

- Frage: Wie sollen in einem gemeinsamen Katalog die Suchresultate priorisiert werden. D.h. welches Resultat hat für den aktuellen User die höchste Relevanz. Diese Frage ist insbesondere bei fehlenden Zusatzinformationen (wie User Login oder zusätzlichen Suchparametern) nicht ganz trivial.
  - Manuelles management? Sortieren nach Anzahl Downloads? Google/SOLR/... für Ideen fragen?
- Des weiteren sollten bei der Datenorganisation die verschiedenen Bezugsformate aggregiert werden können: ein Suchresultat mit "Verfügbar als WMS, WFS, WMTS, SHP, GPKG und natürlich XTF"
- Über eine solche Suche sollten idealerweise auch externe Frontends beliefert werden können mit Text, Icon, Tags, detailliertem Text, Vorschaubild, ...

**Comment by davidoesch** [comment]
habe www.geocat.ch Team angefragt um eine Antwort--sie werden sich melden

**Comment by rarnaud** [comment]
Hallo zusammen

In der **geocat.ch-Suche** kann man filtern durch:
•	Geokataloge (z.B. Kanton Freiburg)
•	Schlüsselwörter (in die erweiterte Suche / z.B. BGDI, …)
•	Facette (z.B. Diensttypen / z.B. Formate)
•	Etc.
Betreffend die Unterscheidung der Kantone, kann man die mit der Katalog-Suche suchen (siehe oben) oder noch mit die «Organisation» Feld in die erweiterte Suche suchen. In beiden Fälle ist die Suche über die Organisation basiert.
Es ist möglich verschiedene Suchkriterien zu kombinieren.
In der Zukunft (normalerweise ca. Ende Jahr werden dann nur Facette dann für die Suche geben (Migration auf GeoNetwork 4).

Alle diese Filtern-Möglichkeiten kann man dann in eine **CSW Abfrage** definieren, was dann wertvoll ist.
Zum Beispiel für:
•	Formate
https://www.geocat.ch/geonetwork/srv/deu/csw?maxRecords=1000&startPosition=1&request=GetRecords&service=CSW&version=2.0.2&resultType=results_with_summary&namespace=xmlns(csw=http://www.opengis.net/cat/csw/2.0.2)&typeNames=csw:Record&constraintLanguage=FILTER&constraint_language_version=1.0.0&elementSetName=full&sortBy=title:A&constraint=%3Cogc:Filter%20xmlns:ogc=%22http://www.opengis.net/ogc%22%3E%3Cogc:PropertyIsEqualTo%3E%3Cogc:PropertyName%3Eformat%3C/ogc:PropertyName%3E%3Cogc:Literal%3EDXF%3C/ogc:Literal%3E%3C/ogc:PropertyIsEqualTo%3E%3C/ogc:Filter%3E
•	Diensttypen
https://www.geocat.ch/geonetwork/srv/deu/csw?maxRecords=1000&startPosition=1&request=GetRecords&service=CSW&version=2.0.2&resultType=results_with_summary&namespace=xmlns(csw=http://www.opengis.net/cat/csw/2.0.2)&typeNames=csw:Record&constraintLanguage=FILTER&constraint_language_version=1.0.0&elementSetName=full&sortBy=title:A&constraint=%3Cogc:Filter%20xmlns:ogc=%22http://www.opengis.net/ogc%22%3E%3Cogc:PropertyIsEqualTo%3E%3Cogc:PropertyName%3Etype%3C/ogc:PropertyName%3E%3Cogc:Literal%3Ebasicgeodata-cantonal%3C/ogc:Literal%3E%3C/ogc:PropertyIsEqualTo%3E%3C/ogc:Filter%3E
•	Kanton/Organisation / Schlüsselwörte
https://www.geocat.ch/geonetwork/srv/deu/csw?maxRecords=1000&startPosition=1&request=GetRecords&service=CSW&version=2.0.2&resultType=results_with_summary&namespace=xmlns(csw=http://www.opengis.net/cat/csw/2.0.2)&typeNames=csw:Record&constraintLanguage=FILTER&constraint_language_version=1.0.0&elementSetName=full&sortBy=title:A&constraint=%3Cogc:Filter%20xmlns:ogc=%22http://www.opengis.net/ogc%22%3E%3Cogc:PropertyIsEqualTo%3E%3Cogc:PropertyName%3E_groupOwner%3C/ogc:PropertyName%3E%3Cogc:Literal%3E54%3C/ogc:Literal%3E%3C/ogc:PropertyIsEqualTo%3E%3C/ogc:Filter%3E

Generell die Namen der Kriterien sind ähnlich, wie die in die URL des Interfaces:
https://www.geocat.ch/geonetwork/srv/fre/catalog.search#/search?_groupOwner=54

Um die ids der Gruppen (z.B. Kantone) kann man das folgende benutzen: https://www.geocat.ch/geonetwork/srv/api/groups

Grüsse

**Reply by m-kuhn** [reply]
Danke für die Antwort.

Ist es auch möglich

 - zu sortieren
   - Beispielsweise Resultate von geoadmin zuorberst, geodienste danach, ...
 - nach diensttypen/formaten zu gruppieren
   - damit ich Duplikate vermeiden kann und für ein Resultat anzeigen kann, wie es verfügbar ist

**Comment by rarnaud** [comment]
Zum filtern kann man den folgenden Parameter zum Beispiel benutzen:
sortBy=title:D

**Comment by hinjud** [comment]
Ich sähe es als Mehrwert, wenn diese Informationen prominenter auf geocat.ch platziert würden. Aktuell muss man sich durch einiges durchklicken, um am richtigen Ort zu landen (z.B. CSW-Infos).

**Comment by hinjud** [comment]
Gerne würde ich der Diskussion einen weiteren Punkt hinzufügen, da aus meiner Sicht das Problem mit der Verfügbarkeit von Daten schon viel früher beginnt: Nämlich mit den vielen politischen und organisatorischen Hürden, die ein Datensatz nehmen muss, bevor er überhaupt in irgendeiner Form schweizweit und zentral zur Verfügung gestellt werden kann. 
Zusammengefasst lautet die Frage: **Wie kann man die gesetzlichen, organisatorische und anderen Hürden schneller und leichter überwinden, um Datensätze mit potentiell breiter Anwendern-/Nutzerbasis schweizweit aufzubereiten und zentral bereitstellen zu können?**

**Comment by rastrau** [comment]
An der GeoUnconference 2 hat eine Gruppe, die sich mit dem Thema des zentralen Zugangs befasst hat, folgende Überlegungen und Lösungsskizze präsentiert:

![1631610797006](https://user-images.githubusercontent.com/4510815/133232736-f053b28b-de8b-404b-bf13-ac51690d7852.jpg)
![1631610796993](https://user-images.githubusercontent.com/4510815/133232761-c05d9596-8741-4042-a719-b8a19ec55156.jpg)

**Comment by rastrau** [comment]
An der GeoUnconference 2 hat die zweite Gruppe, die sich mit dem Thema des zentralen Zugangs befasst hat, folgende Überlegungen angestellt und folgende Lösungsskizze präsentiert:

**Überlegungen zu Vorteilen eines zentralen Zugangs (und offene Fragen):**

![1631610797034](https://user-images.githubusercontent.com/4510815/133232915-2824a439-349c-4729-a51f-4bb9a5cf6ede.jpg)

**Lösungsskizze mit Umsetzung in zwei Phasen:**

![1631610797042](https://user-images.githubusercontent.com/4510815/133233031-a2c8264c-98d5-405a-bc45-eaf5e303f3ee.jpg)

**Anforderungen:**

![1631610797025](https://user-images.githubusercontent.com/4510815/133233041-58152c90-373d-4aa2-9934-9c4f1728e3eb.jpg)

**Comment by davidoesch** [comment]
ein Weitere Möglichkeit
All in der bekannten WMS /WMTS von Behörden erfassen und so vernetzen
Bis  Dato ist die Liste in map.geo.admin.ch
![image](https://user-images.githubusercontent.com/4577727/166458460-f51254d2-0586-4b16-948f-09bdf9ef96e7.png)

über   https://github.com/geoadmin/mf-geoadmin3/blob/master/src/js/ImportController.js ergänzt worden und ist unvollständig

Ein weitere Möglichkeit wäre dann jeder WMS/ jede Quelle so auszubauen das Pro Layer / Pro WMS / Pro Datensatz eine eigentliche Katalog Struktur/ erstellt wird siehe https://github.com/geoadmin/mf-geoadmin3/issues/4668

**Reply by davidoesch** [reply]
übrigens gibt es hierzu einen Ansatz von https://github.com/procrastinatio/geoadminch_configs für die Erstellung externer topics- das es auch erlauben würde , Daten von Diensten einzubinden und als Katalog in map.geo.admin.ch  darzustellen

**Comment by rastrau** [comment]
A large group including Jean-Luc Miserez have discussed how to facilitate access / findability of geodata during GeoUnconference 3. Specifically, the pointed out the importance of:
- Search Engine Optimization (SEO)
- metadata that is – ideally – captured and managed only once
- the steps in finding geodata and how all of these should – ideally – be just one click away from obtaining data
- and more

These are their notes:

![DSC_0959](https://user-images.githubusercontent.com/4510815/170705259-9fecfaac-60cf-43df-9422-a98d773cad41.JPG)

**Comment by rarnaud** [comment]
**A discuter lors de la GeoUnconference du 13.10.2022**

**Constat :**
Les utilisateurs ont besoin d’une plateforme centrale pour rechercher, découvrir, visualiser et télécharger les géodonnées.
Via la saisie directe ou via le moissonnage, geocat.ch permet déjà d’avoir un aperçu de l’ensemble des géodonnées existantes au niveau suisse.
La recherche dans geocat.ch est très dépendante de la qualité de la saisie des géométadonnées.

**Question précise :**
Est-ce que les partenaires de geocat.ch (ex. : cantons, communes, …) envisagent d’implémenter la spécification STAC (SpatioTemporal Asset Catalog) pour leurs géodonnées ?

**Concept :**
L’implémentation dans geocat.ch d’un service de téléchargement des géodonnées basé sur STAC permettrait d’offrir une place centrale au niveau suisse, non seulement pour rechercher les géodonnées (geocat.ch actuel) mais également pour les acquérir (grâce à STAC).
_Dans geocat.ch (V4), on pourrait imaginer que si une ressource a un lien sur un service STAC, il est possible d’afficher le tuilage dans la carte, de sélectionner une emprise spatiale, de sélectionner d’autres paramètres (format, résolution, SRS) et de télécharger les données souhaitées._

**Planification :**
2022-23 : Configuration du SEO pour les métadonnées contenues dans geocat.ch
2023 : Etude de faisabilité pour l’intégration de STAC dans geocat.ch (lien vers le STAC des partenaires)

**Comment by Patabal** [comment]
Résultats du group lors de la 4ème conférence sur le thème "Accès facilité aux données"
Comme il existe déjà beaucoup de plateforme desquelles peuvent être téléchargées les données ou les services, nous avons trouvé qu'il faudrait avoir une meilleure visibilité de toutes ces plateformes existantes (voir liste du groupe thème 7) et mieux connaître le but et l'utilisation de chacune de celles-ci. C'est pourquoi nous avons pensé à une page d'accès en entrée qui permet à l'utilisateur de chercher et filtrer en fonction de ses besoins. De plus, des tutoriaux ainsi qu'un Geo-Wiki seraient intégrés afin d'aider l'utilisateur à comprendre les différents type de données/ services ainsi que de les aider à s'en servir (quelle software, quelle plateforme me permet de lire ces données?). Pour une meilleure gestion de ces données, le principe du "once-only" serait à encourager et aussi de plus lier les données entre elles (voir rediriger) plutôt que de les intégrer.
![GeoUnco#4_Th1_1](https://user-images.githubusercontent.com/115220884/195802591-74c8a1e4-f6ea-48a0-84b5-faf71fc9465d.jpg)
![GeoUnco#4_Th1_2](https://user-images.githubusercontent.com/115220884/195802624-f147776e-1948-4858-8943-f617c45e2445.jpg)

**Comment by davidoesch** [comment]
you can now search 21'000 datasets from swiss gov and drag and drop the desired Layer /Dataset as WMS WMTS WFS directly to your QGIS or ArcGISPro Desktop Client, feedback welcome : test it: https://davidoesch.github.io/geoservice_harvester_poc/
